package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.5.1",
   "description": "LoopBack Connector common code for IBM databases",
   "engines": {
-    "node": ">=6"
+    "node": ">=9"
```

Comment:
I think usually we use even numbers for Node.js versions. Let's change it to "node": ">=8"

Like https://github.com/strongloop/loopback-connector-mysql/blob/master/package.json#L6

See the odd number and even number versions for Node.js in https://nodejs.org/en/about/releases/
and https://nodesource.com/blog/understanding-how-node-js-release-lines-work/

Reply:
@dhmlau said that node 8 reached EOL in Dec 2019...so we still want to reference old node versions?

Reply:
oh, then we should update to node 10
my point was don't use odd number :)